src/dvc_objects/fs/implementations/http.py

```diff
@@ -155,7 +155,7 @@ def isdir(self, *args, **kwargs):
         return False
 
     def isfile(self, *args, **kwargs):
-        return True
+        return self.fs.isfile(*args, **kwargs)
```

Comment:
Let's remove this function altogether, this is implemented on the base class.
https://github.com/iterative/dvc-objects/blob/ebee28733103ac2b13c5207f640394ff03603699/src/dvc_objects/fs/base.py#L156-L157

Reply:
You're right, of course! :bow: I've removed `isfile` from the `HTTPFileSystem` class.